docs/testing/troubleshooting.md

```diff
@@ -1,31 +1,65 @@
 ---
 title: 'Troubleshooting Office Scripts'
 description: 'Debugging tips and techniques for Office Scripts, as well as help resources.'
-ms.date: 12/16/2020
+ms.date: 05/10/2021
 localization_priority: Normal
 ---
 
 # Troubleshooting Office Scripts
 
-As you develop Office Scripts, you may make mistakes. It's okay. We have tools that help find the problems and get your scripts working perfectly.
+As you develop Office Scripts, you may make mistakes. It's okay. You have the tools to help find the problems and get your scripts working perfectly.
+
+## Types of errors
+
+Office Scripts errors fall into one of two categories:
+
+1. Compile-time errors or warnings.
+1. Runtime error.
```

Comment:
```suggestion
2. Runtime errors.
```

Reply:
If you always do "1.", the build process automatically numbers the list. This is great for numbered lists with lots of items because you can add something in the middle and not have to change everything.

That said, this should probably be an unordered list instead.

Reply:
Wow, good to know! Thanks for explaining.